pkg/virt-config/config_test.go

```diff
@@ -152,108 +176,125 @@ var _ = Describe("ConfigMap", func() {
+		table.Entry("when set, GetEmulatedMachines should return the value", "q35, i440*", []string{"q35", "i440*"}),
+		table.Entry("when unset, GetEmulatedMachines should return the defaults", "", strings.Split(virtconfig.DefaultEmulatedMachines, ",")),
 	)
 
-	It("Should return migration config values if specified as json", func() {
-		clusterConfig, _ := testutils.NewFakeClusterConfig(&kubev1.ConfigMap{
-			Data: map[string]string{virtconfig.MigrationsConfigKey: `{"parallelOutboundMigrationsPerNode" : 10, "parallelMigrationsPerCluster": 20, "bandwidthPerMigration": "110Mi", "progressTimeout" : 5, "completionTimeoutPerGiB": 5, "unsafeMigrationOverride": true, "allowAutoConverge": true}`},
+	Context("when migration config is specified as json", func() {
+		It("should return migration config values", func() {
+			clusterConfig, _ := testutils.NewFakeClusterConfig(&kubev1.ConfigMap{
+				Data: map[string]string{virtconfig.MigrationsConfigKey: `{"parallelOutboundMigrationsPerNode" : 10, "parallelMigrationsPerCluster": 20, "bandwidthPerMigration": "110Mi", "progressTimeout" : 5, "completionTimeoutPerGiB": 5, "unsafeMigrationOverride": true, "allowAutoConverge": true}`},
+			})
+			result := clusterConfig.GetMigrationConfig()
+			Expect(*result.ParallelOutboundMigrationsPerNode).To(BeNumerically("==", 10))
+			Expect(*result.ParallelMigrationsPerCluster).To(BeNumerically("==", 20))
+			Expect(result.BandwidthPerMigration.String()).To(Equal("110Mi"))
+			Expect(*result.ProgressTimeout).To(BeNumerically("==", 5))
+			Expect(*result.CompletionTimeoutPerGiB).To(BeNumerically("==", 5))
+			Expect(result.UnsafeMigrationOverride).To(BeTrue())
+			Expect(result.AllowAutoConverge).To(BeTrue())
 		})
-		result := clusterConfig.GetMigrationConfig()
-		Expect(*result.ParallelOutboundMigrationsPerNode).To(BeNumerically("==", 10))
-		Expect(*result.ParallelMigrationsPerCluster).To(BeNumerically("==", 20))
-		Expect(result.BandwidthPerMigration.String()).To(Equal("110Mi"))
-		Expect(*result.ProgressTimeout).To(BeNumerically("==", 5))
-		Expect(*result.CompletionTimeoutPerGiB).To(BeNumerically("==", 5))
-		Expect(result.UnsafeMigrationOverride).To(BeTrue())
-		Expect(result.AllowAutoConverge).To(BeTrue())
 	})
 
-	It("Should return migration config values if specified as yaml", func() {
-		clusterConfig, _ := testutils.NewFakeClusterConfig(&kubev1.ConfigMap{
-			Data: map[string]string{virtconfig.MigrationsConfigKey: `"parallelOutboundMigrationsPerNode" : 10
-"parallelMigrationsPerCluster": 20
-"bandwidthPerMigration": "110Mi"`},
+	Context("when migration config is specified as yaml", func() {
+		It("should return migration config values", func() {
+			clusterConfig, _ := testutils.NewFakeClusterConfig(&kubev1.ConfigMap{
+				Data: map[string]string{virtconfig.MigrationsConfigKey: `"parallelOutboundMigrationsPerNode" : 10
+				"parallelMigrationsPerCluster": 20
```

Comment:
I think this may be the culprit of the unit test failure: you shouldn't wrap lines.